client/src/widgets/qdateeditex.h

```diff
@@ -23,9 +23,10 @@
 #ifndef DATEEDITEX_H
 #define DATEEDITEX_H
 
-#include <QDateEdit>
+#include <KDateComboBox>
+
 #include "fatcrmprivate_export.h"
-class FATCRMPRIVATE_EXPORT QDateEditEx : public QDateEdit
+class FATCRMPRIVATE_EXPORT QDateEditEx : public KDateComboBox
```

Comment:
Please rename the class then. QDateEditEx is a QDateEdit extension I found online.
If we're now extending KDateComboBox then we need a different widget name.
NullableDateComboBox? The null date feature is what this class adds to KDateComboBox, right?